libvast/include/vast/defaults.hpp

```diff
@@ -31,7 +31,7 @@ constexpr size_t max_recursion = 100;
 namespace import {
 
 /// Maximum size for sources that generate table slices.
-constexpr size_t table_slice_size = 1024;
+constexpr size_t table_slice_size = 1 << 16; // 64 ki
```

Comment:
Why not use SI literals here? The header is cheap (no transitive includes) and this becomes *very* clear.

Reply:
I don't want to add a `using namespace vast::si_literals` here, and I can't use the literals without that. I can also just spell the values out, but the literals would only be an option if I did it like this:

```cpp
constexpr auto table_slice_size = []() noexcept -> size_t {
  using namespace vast::si_literals;
  return 64_Ki;
}();
```

Which isn't all that readable and would cause the lambda to be instantiated once per translation unit that includes this file.

Reply:
Uh, so annoying.